tests/openvino/test_modeling.py

```diff
@@ -1891,28 +1894,64 @@ def get_transformer_model_class(self, model_arch):
             from transformers import LlavaNextForConditionalGeneration
 
             return LlavaNextForConditionalGeneration
-        return None
+        return AutoModelForCausalLM
+
+    def gen_inputs(self, model_arch, base_text_prompt, image=None):
```

Comment:
Do you think it would be possible to expose a method like this as a part of an API for working with `OVModelForVisualCausalLM`s? 

It would simplify the demo examples and usage in general. Also, it will come helpful during dataset collection for data-aware compression.

Reply:
I was also thinking that it could help for demos, optimization and validation purposes. No problem if we do it in the follow-up PR.